winpr/libwinpr/clipboard/synthetic_file.c

```diff
@@ -330,9 +330,9 @@ static BOOL do_add_directory_contents_to_list(wClipboard* clipboard, const WCHAR
 		BOOL bRet = FindNextFileW(hFind, &FileData);
 		if (!bRet)
 		{
-			WLog_WARN(TAG, "FindNextFile failed (%" PRIu32 ")", GetLastError());
 			if (ERROR_NO_MORE_FILES == GetLastError())
 				return TRUE;
+			WLog_WARN(TAG, "FindNextFile failed (%" PRIu32 ")", GetLastError());
```

Comment:
One last nit after the last force push: Adjust the commit message (from `Removed unnecessary log message`), since it does not match its actual change anymore.

Maybe something like:

```
winpr/clipboard: Move a log message

ERROR_NO_MORE_FILES is the exit condition to complete a successful
directory traversal. So, don't warn about it.
```

Reply:
ok, updated.